index.html

```diff
@@ -1,14 +1,18 @@
 <!doctype html>
-<html>
+<html prefix="og: http://ogp.me/ns#">
   <head lang="en">
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
-
-    <title>UNIHACK - Sponsorship</title>
-    <meta name="author" content="UNIHACK Inc.">
-    <meta name="description" content="UNIHACK Sponsorship">
-    <meta name="keywords" content="unihack,hackathon,sponsorship">
+    <title>UNIHACK 2019 - Sponsorship</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <meta name="author" content="UNIHACK Incorporated" />
+    <meta name="description" content="UNIHACK returns in March 2019. Sponsor today to get access and recruit top STEM talent in Australia." />
+    <meta name="keywords" content="unihack,hackathon,sponsorship" />
+    <meta name="og:title" content="Sponsor UNIHACK in 2019" />
+    <meta name="og:type" content="website" />
+    <meta name="og:url" content="https://unihack.net/sponsorship/" />
```

Comment:
also the url is going to be sponsorship.unihack.net
it makes sense to isolate the mini site because we have plans to expand it and add a landing page for sponsors too